parm/config/config.vrfy

```diff
@@ -69,33 +69,40 @@ fi
 export ens_tracker_ver=v1.1.15.5
 export HOMEens_tracker=$BASE_GIT/TC_tracker/${ens_tracker_ver}
 
-if [ "$VRFYTRAK" = "YES" ]; then
+if [[ ${machine} == "JET" ]]; then
+   export ens_tracker_ver=v1.1.15.6
+   export HOMEens_tracker=/mnt/lfs4/HFIP/hwrfv3/Jiayi.Peng/ens_tracker.${ens_tracker_ver}
+fi
+
```

Comment:
Revert for now. Will have a follow-up PR to update to a new TC_tracker tag that supports Jet.

Reply:
Reverted the Jet-specific TC tracker info.